### ISSUE TITLE
Modpacks and rewards

### ISSUE THREAD

**Issue by jadedcat** [jakimfett/MMPLv2#32]
Modpack authors can earn rewards on Curse if they so choose, I suggest:

"This mod and any pack containing this mod must remain free to download and available to the general public not limited to an exchange of real world currency. Real world currency includes donations and subscriptions as well as direct payment for access. Access to all components of the mod and any mods combined with it must always be free to the end-user and all financial transactions must remain voluntary and not derived from granting access. "

 To cover allowing pack creators to ship the mod and earn rewards, but not allowing pack creators to charge for access, or server owners to sell parts.

**Comment by Strikingwolf** [comment]
Maybe more of, the end-user must be able to access the pack without exchanging anything in return

The reason I say this is that something like "You must do x, y, and z for me to get access"

Also, we need to make a distinction between public and private packs

**Comment by coolsquid** [comment]
My monetization rule allows monetization as long as it doesn't affect the user experience.

> You are not allowed to use the pack nor the mods commercially, neither directly nor indirectly, from the pack nor its distribution in a way that directly or indirectly affects the user experience.

**Comment by jadedcat** [comment]
Alternative as discussed on IRC:

"Modpack creators may use this mod in a public pack as long as access to the pack is free to everyone. Private packs must also be free of charge. Access cannot be restricted to donators or subscribers. Servers may use this mod as long as they do not charge for access to mods or mod components. They may charge for access to their server, but once on the server all content must be accessible by all players through normal gameplay mechanics"

**Comment by jakimfett** [comment]
@coolsquid my intended purpose for the "complete" license is prevent anyone not the author (or approved distributors) from making money off the mod.

I'm in the process of setting up a "license builder" type site so you can select which of the final options (eg, no monetization) that you'd like to have. It may or may not happen in the near future, as my time to work on side projects is limited by...life in general.

@jadedcat I want to minimize the legalese as much as possible, by reducing specific concepts to general rules, and expressing it in accessible terms. I realize this is a difficult task, and I may not hit the mark on this one. It's a complex topic. I really appreciate you taking the time to discuss this and to open the ticket here.

**Comment by Strikingwolf** [comment]
Using abstractions to minimize legalese may help. That would take some declarations at the beginning of the license. Thinking about it in programming terms (as that's how I think) it's a lot easier to understand the abstract form, and that could help the average person understand the license

**Comment by jadedcat** [comment]
Well in simpler terms:

Modpack use allowed if:

- No real world money is needed to download the pack
- No subscription or donation is needed to access the download
- If the pack is public the modlist must be publicly available
- If the pack is private, access to the pack cannot be based on donations/subscriptions or payment

Server use allowed if:

- Any subscription using real world money is for server access only
- Once a player enters the server all mod components must be available to everyone equally
- Servers may not use the mod if they trade real world money for in game money

**Comment by Strikingwolf** [comment]
Cover server use subscription and donations. For example Patreon servers are allowed, but they cannot give more in-game rewards for higher donators

**Comment by jakimfett** [comment]
@jadedcat Thanks, that's a solid jumping off point for a revision of the modpack permissions. The server use bit is all covered under Mojang's license, so I don't think it's necessary to add those clauses.

@Strikingwolf that's already covered by the Mojang EULA, and is thus unnecessary to call out specifically.
